apps/web/src/components/Notifications/NotificationUserList.tsx

```diff
@@ -1,17 +1,18 @@
-import { Suspense } from 'react';
+import { Suspense, useCallback } from 'react';
 import { useLazyLoadQuery } from 'react-relay';
 import { graphql } from 'relay-runtime';
 import styled from 'styled-components';
 
 import { HStack, VStack } from '~/components/core/Spacer/Stack';
-import { USERS_PER_PAGE } from '~/components/NotificationsModal/constants';
-import { NotificationUserList } from '~/components/NotificationsModal/NotificationUserList/NotificationUserList';
-import { NotificationUserListTitle } from '~/components/NotificationsModal/NotificationUserListTitle';
+import { USERS_PER_PAGE } from '~/components/Notifications/constants';
+import { NotificationUserList } from '~/components/Notifications/NotificationUserList/NotificationUserList';
+import { NotificationUserListTitle } from '~/components/Notifications/NotificationUserListTitle';
 import { BackButton } from '~/contexts/globalLayout/GlobalNavbar/BackButton';
-import { useModalActions } from '~/contexts/modal/ModalContext';
+import { useDrawerActions } from '~/contexts/globalLayout/GlobalSidebar/SidebarDrawerContext';
 import { NotificationUserListModalQuery } from '~/generated/NotificationUserListModalQuery.graphql';
 
 import Loader from '../core/Loader/Loader';
+import { Notifications } from './Notifications';
 
 type NotificationUserListModalProps = {
```

Comment:
this is the component that shows the full list of users who viewed your gallery when you click on that notification.

main change here is to show the Notifications drawer upon clicking back instead of closing a modal.